.github/workflows/Tests.yml

```diff
@@ -8,25 +8,34 @@ on:
 
 jobs:
   test:
+    # Use matrix.test.name here to avoid it taking up the entire window width
+    name: test ${{matrix.test.name}} (${{ matrix.os }}, ${{ matrix.version }}, ${{ matrix.arch }}, ${{ matrix.num_threads }})
     runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.version == 'nightly' }}
 
     strategy:
       fail-fast: false
       matrix:
-        test-args:
+        test:
           # Run some of the slower test files individually. The last one catches everything
           # not included in the others.
-          - "essential/ad.jl"
-          - "mcmc/gibbs.jl"
-          - "mcmc/hmc.jl"
-          - "mcmc/abstractmcmc.jl"
-          - "mcmc/Inference.jl"
-          - "experimental/gibbs.jl"
-          - "mcmc/ess.jl"
-          - "--skip essential/ad.jl mcmc/gibbs.jl mcmc/hmc.jl mcmc/abstractmcmc.jl mcmc/Inference.jl experimental/gibbs.jl mcmc/ess.jl"
+          - name: "essential/ad"
+            args: "essential/ad.jl"
+          - name: "mcmc/gibbs"
+            args: "mcmc/gibbs.jl"
+          - name: "mcmc/hmc"
+            args: "mcmc/hmc.jl"
+          - name: "mcmc/abstractmcmc"
+            args: "mcmc/abstractmcmc.jl"
+          - name: "mcmc/Inference"
+            args: "mcmc/Inference.jl"
+          - name: "experimental/gibbs"
+            args: "experimental/gibbs.jl"
+          - name: "mcmc/ess"
+            args: "mcmc/ess.jl"
+          - name: "everything else"
+            args: "--skip essential/ad.jl mcmc/gibbs.jl mcmc/hmc.jl mcmc/abstractmcmc.jl mcmc/Inference.jl experimental/gibbs.jl mcmc/ess.jl"
         version:
-          - '1.7'
```

Comment:
Do we want to remove this, or rather replace with 1.9? My thinking would be that we would fix one to 1.9 and let the other one be 1 and have it float up as new Julia releases come.

Reply:
I was kind of unsure about this. I feel like it must be worth explicitly testing on 1.10, but that would lead to us testing all three of 1.9 (min bound), 1.10 (LTS), and 1 (newest version), which might be a bit much.

Reply:
But if we had to cut one from the three, I'd cut 1.10, and assume that if it works on 1.9 and 1 then it should work on 1.10 (kind of a sandwich theorem).